src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="imac-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="imac-card__image"
+      >
+      <p class="imac-card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
```

Comment:
```suggestion
      <h2 class="imac-card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
```
Це можна зробити заголовком